<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Pays</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>871</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>6</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajouter les Pays</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>831</width>
        <height>391</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajout</string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>110</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>nom pays</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>230</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>100</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Afficher les Pays</string>
     </attribute>
     <widget class="QTableView" name="tab_Pays">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>711</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>370</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>afficher</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string> Supprimer les pays</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Identifiant</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_supprimer">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>230</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>120</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Modifier</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>150</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom pays</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Donner l id de pays à modifier</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="n_pays">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>150</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_modifier">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>300</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>modifer</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_3">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>100</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Classement par Medailles</string>
     </attribute>
     <widget class="QTableView" name="TAB_Classement">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>841</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>400</y>
        <width>211</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Afficher</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Statistique </string>
     </attribute>
     <widget class="QTableView" name="tab_r">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>91</y>
        <width>411</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>400</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Afficher</string>
      </property>
     </widget>
     <widget class="QTableView" name="tab_h">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>91</y>
        <width>381</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>350</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Afficher</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>291</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre de médailles rapporté à la population</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>40</y>
        <width>261</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>les pays qui ont les plus nombres d habitants</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>Declarer forfait</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>110</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>donner l  id du pays</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_4">
      <property name="geometry">
       <rect>
        <x>352</x>
        <y>110</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_declarer">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>370</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>declarer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>190</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>raison</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_5">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>190</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>150</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>reclamation</string>
     </attribute>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Donner le nom du pays</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_6">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>80</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>370</y>
        <width>211</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Passer une reclamation</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>149</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>raison</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_7">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>150</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_9">
     <attribute name="title">
      <string>Alerte</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>691</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>953</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGestion_des_pays">
    <property name="title">
     <string>Gestion des pays</string>
    </property>
   </widget>
   <addaction name="menuGestion_des_pays"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
